README.md

```diff
@@ -272,6 +272,7 @@ following projects all build on top of Dependencies:
 
   * [Dependencies Additions](https://github.com/tgrapperon/swift-dependencies-additions): A
     companion library that provides higher-level dependencies.
+  * [Dependencies Extras](https://github.com/arasan01/swift-dependencies-extras): Macros that make swift-dependencies more useful
```

Comment:
Definitely open to adding this, but given that we already have `swift-dependencies-additions` being a library extending `swift-dependencies` with higher-level dependencies, have you thought about naming the library more specifically related to the macros it offers?

I know renaming a library is a tall order, so no worries if you're happy with the way things are. Just lemme know and I'll merge when you're ready :)